mailbox/store/src/main/java/org/apache/james/mailbox/store/json/event/dto/UserDataTransferObject.java

```diff
@@ -0,0 +1,50 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.store.json.event.dto;
+
+import java.util.Optional;
+
+import org.apache.james.core.Domain;
+import org.apache.james.core.User;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class UserDataTransferObject {
+
+    @JsonProperty()
+    private String localPart;
```

Comment:
Prefer @JsonCreator and final fields

Reply:
Jackson requires a default constructor: `UserDataTransferObject()` which take no parameter have to be declared. Then I could not put final to those fileds.